jovian/jovian_nb_ext/main.js

```diff
@@ -617,16 +616,11 @@ define([
               }
             });
           };
-          show("#if_new", { "#nb_opt": true, "#project_id_box": false });
-          show("#if_git", { "#git_msg_box": true });
+          show("#if_new", { "#project_id_box": false });
           $("#if_new").change(() => {
-            show("#if_new", { "#nb_opt": true, "#project_id_box": false });
+            show("#if_new", { "#project_id_box": false });
             project_id_helper();
           });
-          $("#if_git").change(() => {
-            show("#if_git", { "#git_msg_box": true });
-            git_message_helper();
```

Comment:
You can retain this, but listen to change in #project_msg_box and update the value of #git_msg_box same as #project_msg_box.

Also add placeholder for #project_id_box as `auto unless specified`

Reply:
i'll merge this now. you can do the above changes in a new PR